dapr-spring-boot-starter/src/main/java/io/dapr/springboot/DaprAutoConfiguration.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Dapr's Spring Boot AutoConfiguration.
+ */
+@Configuration
+@ConditionalOnWebApplication
+@ComponentScan("io.dapr.springboot")
```

Comment:
Is this the only package to be scanned?

Reply:
Yes, it's the package where the Spring beans `DaprBeanPostProcessor` and `DaprController` locate, they are the beans provided by dapr to be scanned by spring boot.